apis/nucleus/src/components/listbox/hooks/use-style.js

```diff
@@ -0,0 +1,43 @@
+function getValue(object, path, fallback) {
+  if (object === undefined || path === undefined) {
+    return fallback;
+  }
+
+  const steps = path.split('.');
+
+  let scoped = object;
+
+  for (let i = 0; i < steps.length; ++i) {
+    const step = steps[i];
+
+    if (scoped[step] === undefined) {
+      return fallback;
+    }
+
+    scoped = scoped[step];
+  }
```

Comment:
If may be self-explanatory but you could add a short comment about what the function does, and why you return fallback on `undefined`.

Reply:
Actually, could you use this instead?
```
utils.getValue(object, 'path', fallback);
```
```

Reply:
`utils.getValue` is in the `conversion` package, right? But I was thinking the same. Just just introduces this helper function one more time in nebula. Don't know the best/preferred way of sharing it between packages within nebula.js though...

Reply:
Ok, yeah can check if it's possible to re-use somehow then from `api/conversion/src/utils.js` @CostInCode 
Otherwise, add it to a location where can easily import it so we don't duplicate it several times inside Nebula.